app/views/case_contacts/_case_contact.html.erb

```diff
@@ -43,11 +43,10 @@
     <% end %>
   </td>
   <td>
-    <%# TODO What text for tooltip? %>
     <% if Pundit.policy(current_user, contact).update? %>
-      <div class="add-disallow-edit-tooltip" data-toggle="tooltip" title="Disallow edit contact after end of quarter">
```

Comment:
Hmm, I don't want to lose this tooltip... maybe we can put it back with a "?"/icon instead of "Edit"
I will make a github issue for this and post it here

Reply:
https://github.com/rubyforgood/casa/issues/1072